### PR TITLE
Make consent respond to textScaleFactor

### LINKED ISSUE #72: [BUG] Text on "Informed Consent" screen does not honor textScaleFactor
**Describe the bug**
The main text in the "Informed Consent" does not honor textScaleFactor. It is always the same size.

**To Reproduce**
0. Delete current app session if one exists
1. Set textScaleFactor to 3 with MediaQuery
2. Click "Check up on your health" on the main screen
3. Observe that the consent text is not enlarged

**Expected behavior**
Consent text scales with textScaleFactor.

**Smartphone (please complete the following information):**
 - Device: Any
 - OS: Any
 - App version: 0.1.0+1

### PR DESCRIPTION
**Pull request by gspencergoog**
This makes the consent screen (and the consent declined screen) respect text scale factor, and also puts the buttons at the bottom in a `Wrap` instead of a `Row`, so that at large text sizes it doesn't overflow.

Fixes #72 

Screenshots:
Initially:
image

After not agreeing:
image

After going back:
image

### CODE REVIEW

**Comment by clocksmith** [comment]
LGTM, but not sure about the pubspec.lock changes

File: ios/Podfile.lock [hunk 1]
```diff
@@ -66,4 +66,4 @@ SPEC CHECKSUMS:
 
 PODFILE CHECKSUM: c1f56ec578e7dc933512aa9b9b86b80eb2a5b47d
 
-COCOAPODS: 1.9.1
+COCOAPODS: 1.8.4
```
Comment (dnfield) [inline ios/Podfile.lock:69]: Probably want to avoid downgrading cocoapods here.

Comment (gspencergoog) [reply ios/Podfile.lock:69]: Yeah, that keeps leaking in. I thought that the decree was that we were using the stable branch, right?

**Review by dnfield — DISMISSED** [review]
LGTM other than the podfile change, although we can probably land with that and just have it bounce back again later.

**Comment by dnfield** [comment]
Looks like test(s) are failing on CI.

File: ios/Podfile.lock (hunk 1 above)
Comment (dnfield) [reply ios/Podfile.lock:69]: Yes - but I believe this is tied to your local `pod` version rather than to Flutter's version, right?

**Comment by gspencergoog** [comment]
Weird. There's some kind of state leakage between tests if I instantiate the consent step more than once (if I change the order of the tests, the failing test is different).

Solved it by merging my test into an existing one, but probably should figure that out eventually.

**Review by dnfield — APPROVED** [review]

File: ios/Podfile.lock (hunk 1 above)
Comment (gspencergoog) [reply ios/Podfile.lock:69]: Ahh, so that's it.  Upgrading my cocoapods now!

**Pull request merged**